src/tests/ftest/util/mpio_utils.py

```diff
@@ -69,15 +66,18 @@ def run_romio(self, basepath, hostlist, romio_test_repo):
                 romio_test_repo --built romio test directory
         """
 
-        # environment variables only to be set on client node
-        env_variables = ["export CRT_ATTACH_INFO_PATH={}/install/tmp/" \
-                             .format(basepath),
-                         "export MPI_LIB=''", "export DAOS_SINGLETON_CLI=1"]
-
-        # setting attributes
-        cd_cmd = 'cd ' + romio_test_repo
-        test_cmd = './runtests -fname=daos:test1 -subset -daos'
-        run_cmd = cd_cmd + ' && ' + test_cmd
+        env = EnvironmentVariables()
+        env["CRT_ATTACH_INFO_PATH"] = "{}/install/tmp/".format(basepath)
```

Comment:
Isn't it that with PMIx gone these CRT_ATTACHE_INFO_PATH and DAOS_SINGLETON_CLI env variables on client side should also be removed?
Client should be using the daos_agent.yml file under install/tmp should be used instead for communication.

Reply:
Done.

Reply:
May be the changes still need to be pushed. I still see the env vars.

Reply:
Yeah, something didn't get pushed in the last update.  Should be gone now.